scripts/sct_deepseg_sc.py

```diff
@@ -155,8 +168,8 @@ def main():
     im_image = Image(fname_image)
     # note: below we pass im_image.copy() otherwise the field absolutepath becomes None after execution of this function
     im_seg, im_image_RPI_upsamp, im_seg_RPI_upsamp, im_labels_viewer, im_ctr = deep_segmentation_spinalcord(
-        im_image.copy(), contrast_type, ctr_algo=ctr_algo, ctr_file=manual_centerline_fname,
-        brain_bool=brain_bool, kernel_size=kernel_size, remove_temp_files=remove_temp_files, verbose=verbose)
+        im_image.copy(), contrast_type, ctr_algo = ctr_algo, ctr_file = manual_centerline_fname,
+        brain_bool = brain_bool, kernel_size = kernel_size, remove_temp_files = remove_temp_files, verbose = verbose)
```

Comment:
i think in PEP8 there is no space for assigning arguments in function calls (please check)

Reply:
I checked and you are correct. [PEP8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements) states: 

> Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter

Changes have been made in commit [9244160](https://github.com/neuropoly/spinalcordtoolbox/commit/924416036a04d4e367b65d14cb513fe577162036)